avm/res/desktop-virtualization/scaling-plan/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,42 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.sp-${serviceShort}-rg'
```

```suggestion
param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.scalingplans-${serviceShort}-rg'
```

If the name doesn't happen to get too long (which it shouldn't) I recommend not using abbreviations.